soc/xtensa/intel_adsp/cavs_v15/Kconfig.defconfig.series

```diff
@@ -66,15 +66,9 @@ if LOG
 config LOG_PRINTK
 	default y
 
-config LOG_BACKEND_RB
```

Comment:
Just a notice. That ring buffer backend was supposed to be general one which can be used for other boards.

Reply:
But it speaks a protocol specific to the audio hardware on Intel CPUs.  I mean, sure, it's possible someone else might choose to use that, but even then it wouldn't be wrong to call it the "Intel ADSP protocol".

Also, removing the ring buffer data structure from the equation shrinks the SOF code size (which doesn't otherwise use it) by a few hundred bytes, which has value too.